lambda_functions/healthcheck/healthcheck.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+import boto3
+import requests
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel(logging.ERROR)
+
+def check_web_url_health(url, expected_string, env):
+    """
+    check_web_url_health is a function that calls the url from the event
+    and checks if the status code is 200 and the expected string exists on that url.
+
+    :param url: Url of url to be checked
+    :type url: string
+    :param expected_string: String of text to be checked
+    :type expected_string: String
+    :param env: This is for us to distinguish the message between the dev, preprod and prod environments
+    :type env: string
+    :raises logger.error: will log an error to the lambda log group
```

Comment:
I don't think raise should be used for logger.error

Reply:
removed :)